docs/hw/overview.md

```diff
@@ -9,19 +9,14 @@ The top of the robot contains three buttons which can all be overloaded by a ROS
 The power button features a ring of six RGB LEDs for indication.
 
 ![Create® 3 from its above-rear-left, with the top cover and cargo bay removed.](data/rear_iso.jpg "Robot Rear")
-The faceplate and cargo bay of the robot can be removed for quick prototyping.
-The faceplate is removed by rotating it on the center axis of the robot using the thumb rests, while the cargo bay is removed by simply pulling on the handle.
-The locating pins on the top cover are used to install and retain the faceplate.
-![How to Remove or Attach Create® 3 Faceplate](data/remove_faceplate.svg "How to Remove Create® 3 Faceplate")
-
-There are two cable passthroughs.
-The rear passthrough is good for quick prototyping while the one that penetrates the top cover and faceplate is useful to keep wires within the radius of the robot.
+The faceplate and cargo bay of the robot feature a regular hole pattern for attaching payloads and can be removed without tools for quick prototyping.
+There are two cable passthroughs: one on the top edge of the cargo bay which is good for quick prototyping and one that penetrates the top cover and faceplate which is useful for keeping wires within the radius of the robot.
```

Comment:
Please add a link to the Mechanical page at the end of this paragraph ("More information on these features can be found at..." or similar, mirroring line 21/16.

Reply:
Done.